### PR TITLE
LPS-180512  | Test Fix | Master

### PR DESCRIPTION
**Pull request by sirfabios**
https://issues.liferay.com/browse/LPS-180512

### CODE REVIEW

**Comment by sirfabios** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/paths/ProcessBuilderListView.path [hunk 1]
```diff
@@ -57,7 +57,7 @@
 </tr>
 <tr>
 	<td>CLEAR_BUTTON_PROCESS_BUILDER</td>
```
Comment (Luizcarlosqueiroz) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/paths/ProcessBuilderListView.path:59]: This Path does the same thing as this other: ManagementBar#CLEAR_FILTER.
I think we could remove this CLEAR_BUTTON_PROCESS_BUILDER path, as it's only used in this failing test, and replace by this other path.

**Review by Luizcarlosqueiroz — CHANGES_REQUESTED** [review]
Hey @sirfabios !
I left a comment there, but basically this test uses a Path from a component (management bar) that is not ours. So it's best to use the Path from the team that actually owns the component, and who will maintain this Path.

**Comment by sirfabios** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/paths/ProcessBuilderListView.path (hunk 1 above)
Comment (sirfabios) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/paths/ProcessBuilderListView.path]: Done

**Comment by sirfabios** [comment]
ci:forward

**Pull request closed without merging**